src/pybind/mgr/debug/module.py

```diff
@@ -0,0 +1,64 @@
+from mgr_module import CLIReadCommand, HandleCommandResult, MgrModule, Option
+from threading import Event
+from typing import cast, Any, Optional, TYPE_CHECKING
+import errno
+import epdb
```

Comment:
yeah, AFAIR it was something like:

```
try:
    imoprt module_foo
except:
    print(sys.path)
```
In my case the import was failing because `vstart` doesn't import user's python modules (it only uses system ones).